ecc/blocks/ecc-dashboard/ecc-dashboard.js

```diff
@@ -666,6 +666,10 @@ function filterData(props, config, query) {
   props.filteredData = props.data.filter((e) => {
     const defaultLocale = e.defaultLocale || Object.keys(e.localizations)[0] || 'en-US';
     const eventTitle = getAttribute(e, 'title', defaultLocale);
+    if (!eventTitle) {
+      window.lana?.log(`event Title is not defined ${e.eventId}`);
+      return false;
+    }
     return eventTitle.toLowerCase().includes(q);
```

Comment:
Technically this change should be achievable with only adding a ? between `eventTitle` and `.toLowerCase()`. But I understand that's probably not to everyone's liking.

Reply:
@luckyluke-adobe let me know your thoughts on this. Otherwise we can merge it.